api/location_management.py

```diff
@@ -41,3 +43,48 @@ def get_schedule_data(location, date):
                 location_param  = get_id(location), 
                 date_param      = date)
             ).json()['Result']
+
+def get_event_data(restaurant: str) -> dict:
+    '''
+    Given a valid location and date,
+    perform get request, then parse the HTML code for the event_json using BeautifulSoup 4
+    '''
+    url = 'https://uci.campusdish.com/LocationsAndMenus/'
+    if(restaurant == 'Anteatery'):
+        url += 'TheAnteatery'
+    else:
+        url += restaurant
+    html = requests.get(url).text
+
+    soup = bs(html, 'html.parser')
+    table = soup.findAll('td', {'style': ['padding: 0.75pt; border-width: 0.75pt; border-right-style: dashed; border-right-color: #999999; border-bottom-style: dashed; border-bottom-color: #999999; text-align: left;',\
+                                        'padding: 0.75pt; border-width: 0.75pt; border-right-style: dashed; border-right-color: #999999; border-bottom-style: dashed; border-bottom-color: #999999; text-align: center;',\
+                                        'border-right: 0.75pt dashed #999999; border-bottom: 0.75pt dashed #999999; text-align: left;',\
+                                        'padding: 0.75pt; border-right: 0.75pt dashed #999999; text-align: left;',\
+                                        'padding: 0.75pt; border-bottom: 0.75pt dashed #999999; text-align: left;']})
+    entries_list = list()
+
+    for i in table:
```

Comment:
This would be a good place to use the built-in `map` function instead of a loop.

Reply:
map function is used in the latest commit